modules/ducktests/tests/ignitetest/tests/client_in_out_test.py

```diff
@@ -0,0 +1,173 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains client tests
+"""
+import time
+from ducktape.mark import parametrize
+from ignitetest.services.ignite import IgniteService
+from ignitetest.services.ignite_app import IgniteApplicationService
+from ignitetest.services.utils.control_utility import ControlUtility
+from ignitetest.services.utils.ignite_configuration.cache import CacheConfiguration
+from ignitetest.services.utils.ignite_configuration import IgniteConfiguration
+from ignitetest.utils import ignite_versions
+from ignitetest.utils.ignite_test import IgniteTest
+from ignitetest.utils.version import DEV_BRANCH, V_2_8_1, IgniteVersion
+
+
+# pylint: disable=W0223
+class ClientTest(IgniteTest):
+    """
+    cluster - cluster size
+    CACHE_NAME - name of the cache to create for the test.
+    PACING - the frequency of the operation on clients (ms).
+    JAVA_CLIENT_CLASS_NAME - running classname.
+    client_work_time - clients working time (s).
+    iteration_count - the number of iterations of starting and stopping client nodes (s).
+    static_clients - the number of permanently employed clients.
+    temp_client - number of clients who come log in and out.
+    """
+
+    CACHE_NAME = "simple-tx-cache"
+    PACING = 10
+    JAVA_CLIENT_CLASS_NAME = "org.apache.ignite.internal.ducktest.tests.start_stop_client.IgniteCachePutClient"
+
+    @ignite_versions(str(V_2_8_1), str(DEV_BRANCH))
+    @parametrize(cluster=7,
+                 static_clients=2,
+                 temp_client=3,
+                 iteration_count=3,
+                 client_work_time=30)
+    # pylint: disable=R0913
+    def test_ignite_start_stop_nodes(self, ignite_version,
```

Comment:
To enable run the test in parallel with others please use @cluster decorator. If user wants to run experiments with different number of nodes, the one must comment the decorator in feature branch.

Actually we need a flag for ducktape to ignore this decorator if needed. I will make a ticket for that.

Reply:
ok. I found your ticket thank you!